packages/protocol/script/layer1/based/DeployProtocolOnL1.s.sol

```diff
@@ -314,17 +314,17 @@ contract DeployProtocolOnL1 is DeployCapability {
                 )
             );
         }
-        address helper = address(new InboxHelper());
+        address simpleCodec = address(new SimpleCodec());
         address tempFork =
-            address(new DevnetShastaInbox(proofVerifier, whitelist, bondToken, helper));
+            address(new ShastaDevnetInbox(simpleCodec, proofVerifier, whitelist, bondToken));
```

Comment:
I see `ShastaDevnetInbox` is from `InboxOptimized2`, and `SimpleCodec` is not aligned with the encoding / decoding methods in `InboxOptimized2`?